vasudeva-ram/Julia-SSJ
# packaging up

* moved files into a package structure
* deleted `Manifest.toml` and added to gitignore
* added unit test structure
* removed some seemingly unused dependencies

how to test:

* open vscode in package root
* `] activate .` if not already in `SSJ` environment
* `] instantiate`
* `] test` to run tests (not very informative so far, but should run)

to develop code:

```julia
using Revise
using SSJ

SSJ.main()  # runs main()

# edit main() in VScode

SSJ.main() # reflects changes made in source code (no manual reloading of code necessary)
```
